storm-multilang/python/src/main/resources/resources/storm.py

```diff
@@ -29,6 +30,26 @@
 json_encode = lambda x: json.dumps(x)
 json_decode = lambda x: json.loads(x)
 
+# scheduled timer running job at fixed rate (note: GIL!)
```

Comment:
Thanks @dan-blanchard ,
I experimented about python's GIL just now, and with python 2.7.6 in OSX I found that other thread can hold CPU more than 1 sec when timer is expired at that time. 
https://gist.github.com/HeartSaVioR/34d90cdd6af906e72935

Actually I wasn't affected this issue during I was working with Python cause it was I/O intensive job, and seems like it isn't same to CPU intensive job.

Default tick time is somewhat very long. I found one document which says tick time is about ~6.5 secs, which doesn't meet our requirement.

GIL is a major problem, and maybe I should check it with Ruby.